src/views/home/Discussion.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <div>
     <div class="discussion">
+      <div class="py-2 new">
```

Comment:
Bu sayfa zaten tartışma detay sayfası oluğu için buradan bir tartışma oluşturulamamalı.Tartışma oluşturma tuşu topluluk detay sayfasında bulunmalı